assets/scss/components/_load-modal.scss

```diff
@@ -0,0 +1,50 @@
+.load-modal {
+
+	&__wrap {
+		border-radius: 25px;
+	    width: 426px;
+	    height: 465px;
+	    padding: 45px 80px;
+	}
+	&__button--wrap {
+		margin: 0 auto;
+    	text-align: center;
+    	 margin-bottom: 20px;
+	}
+	&__button {
+		padding-right: 60px;
+	    padding-left: 60px;
+	    font-size: 20px;
+	    font-weight: 200;
+	    margin-top: 20px;
+
+	}
+	&__link {
+		color: #059AC0;
```

Comment:
Use variables for all colors.  Makes it easier when you have to reuse colors, or if you change the color - easier to do it sidewide rather than some big search and replace.